### ISSUE TITLE
Reclassify ARIA failures under 4.1.2

### ISSUE THREAD

**Issue by dylanb** [dequelabs/axe-core#986]
<pre><code>
axe-core version: all
axe-webdriver, extension or other integration version: all
</code></pre>

The following rules should be classified only under WCAG 2 S.C. 4.1.2

```
aria-allowed-attr
aria-required-attr
aria-roles
aria-valid-attr
aria-valid-attr-value
```
